strapi/strapi
# fix: remove double conversion that causes date shift

### What does it do?

removed the `convertLocalDateToUTCDate` function which caused double conversion and date shift from the client side (check the video posted in #22023 ), the component now preserves the actual calendar date the user selected and eliminates timezone-related shifts (same behaviours as in V4)

### Why is it needed?

by default, `Date` object will be created in the local time zone of the user's browser, but dates should be kept consistent in UTC at all times

### How to test it?

- launch `examples/getStarted`
- change your device's timezone to PST (Los Angeles for example )
- go to content type builder, create a collection with a `date` field
- go to content manager, create a new entry
  please check:
   1. When selecting a date from a date picker, the date displayed in the input is the selected value.
   2. Without selecting the date, if the datePicker loses focus (onBlur), the date shown in the input is not changing.
 - save the new entry, and change the timezone on your device again (to Singapore for example)
 - the displayed date doesn't change its value
-  change back the timezone and verify the value again.

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/issues/22023

DX-1788
